examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -2168,7 +2168,7 @@ OTCALL
 otSendActiveSet(
     _In_ otInstance *aInstance, 
     const otOperationalDataset *aDataset, 
-    const uint8_t *aTlvs,
+    uint8_t *aTlvs,
```

Comment:
Here `otSendActiveSet()` function declaration in `openthread.h` below. I think it should keep the consistent with the declaration. Previously Appveyor check detects this issue and fail to build on Win platform, so remove the `const` modifiers here.